tests/system/providers/amazon/aws/redshift/example_redshift_cluster.py

```diff
@@ -28,7 +28,11 @@
     RedshiftResumeClusterOperator,
 )
 from airflow.providers.amazon.aws.sensors.redshift_cluster import RedshiftClusterSensor
+from airflow.utils.trigger_rule import TriggerRule
+from tests.system.providers.amazon.aws.utils import set_env_id
 
+ENV_ID = set_env_id()
+DAG_ID = 'example_redshift_cluster'
```

Comment:
I can't comment below, but this should be used in the "with DAG" block:

with DAG(
    dag_id=DAG_ID,
    ....
    ) as dag:

Reply:
resolved